docs/admin/Installation/Cloud-Providers/aws.md

```diff
@@ -0,0 +1,206 @@
+# AWS
+## Overview
+By supporting AWS cloud providers, CCX offers a comprehensive platform for deploying and managing database instances as part of a DBaaS (Database as a Service) solution. The integration with AWS allows users to leverage AWS's infrastructure management, enabling automated provisioning, scaling, and maintenance of databases with a high level of agility and flexibility.
+
+CCX interacts with AWS APIs to automate the creation, configuration, and management of database instances, minimizing manual intervention and reducing potential for configuration errors.
+
+## Requirements
+To fully integrate CCX with AWS for DBaaS, the following resources and API access are necessary.
+
+##  Prerequisites
+
+### Required Resources
+For proper functioning with AWS, CCX needs access to the following resources:
+
+#### Compute Resources (EC2 Instances):
+CCX requires permission to create and manage EC2 instances within AWS. These instances will host database instances and are provisioned dynamically based on workload demands.
+
+#### Public and Private IP Addresses:
+CCX should have the ability to allocate public and private IP addresses as needed for database instances. Public IPs allow external access, while private IPs ensure secure internal communication.
+
+#### Security Groups:
+CCX needs to create and manage security groups to define access control to the EC2 instances. This includes specifying allowed ports, protocols, and IP ranges for secure communication.
+
+#### Volume Management (EBS):
+CCX requires the ability to create and attach Elastic Block Storage (EBS) volumes to EC2 instances for database storage. Configurable volume sizes allow users to tailor storage capacity to their specific database needs.
```

Comment:
I think it would be nice to have this in AWS IAM format (where you are granting these permissions)

Reply:
I searched it in jira, confluence but yet i coudn't find the fine grained roles. i need backend help if its already been documented. So that I can add it here

Reply:
done